node/package.json

```diff
@@ -29,11 +29,11 @@
     "mockery": "^1.7.0",
     "q": "^1.1.2",
     "semver": "^5.1.0",
-    "shelljs": "^0.3.0",
     "uuid": "^3.0.1"
   },
   "devDependencies": {
     "mocha": "5.2.0",
+    "shelljs": "^0.3.0",
```

Comment:
Can we get rid of this?

Reply:
We still use it in some of our build process. I think that would be the next step, but that feels secondary to the primary goal of getting rid of any functional impact from shelljs (and is somewhat orthogonal).

To be clear, I want to get rid of it, just also want to keep PRs as focused/manageable as possible.

Reply:
Makes sense, can we create an issue to track it.

Reply:
Added #523 to track this